developer_guide/installation/system_requirements/system_requirements.rst.inc

```diff
@@ -50,6 +50,10 @@ The web server will also need the following libraries and modules:
 
   function `exec()` is required in order to handle data import/export, check if it hasn't been disabled in your php.ini.
 
+.. warning::
+
+  To avoid a security vulnerability please be sure that you use PHP >= 5.6.24, please have a look to https://github.com/FriendsOfPHP/security-advisories/issues/259#issuecomment-369274526 and https://github.com/php/php-src/commit/0e2447cd11f4b72257e5d2609f923177e9736c3c
```

Comment:
vulnerability, please make sure you are using [...]. More details available at: [...]